README.md

```diff
@@ -36,14 +36,32 @@ following to your keymap to map `Ctrl-x Ctrl-f` to open the dialog:
 ## Settings:
 
 <dl>
-  <dt>Case-sensitive auto-completion</dt>
+  <dt>Case-sensitive Auto-completion</dt>
```

Comment:
The capital "A" seems like it contradicts the lowercase "f" and "i" on the next time…